server/index.js

```diff
@@ -1,23 +1,14 @@
 const express = require('express');
 const app = express();
 const port = 3004;
-const mysql = require('mysql');
-
-const connection = mysql.createConnection({
-  host: 'localhost',
-  user: 'root',
-  password: 'password!',
-  database: 'fec_reviews'
-})
-
+const db = require('../database/index.js');
 
 app.get('/api', (req, res) => {
-  connection.connect();
-  connection.query('SELECT * FROM reviews', (err, result) => {
+  db.query('SELECT * FROM reviews', (err, result) => {
     if (err) throw err;
     res.send(result);
   });
-  connection.end();
+  db.end();
```

Comment:
I think this line might cause problems. The .end call might happen before the query is finished.